pages/plans.tsx

```diff
@@ -0,0 +1,131 @@
+export const plansCardData = [
```

Comment:
let's declare this inside the getServerSieProps function instead of at the top of the file